src/app/components/login/InviteNewAccount.js

```diff
@@ -51,7 +30,7 @@ const InviteNewAccountComponent = ({ user }) => {
     if (checkedTos) {
       const onFailure = (errors) => {
         const errorMessage = getErrorMessageForRelayModernProblem(errors) || <GenericUnknownErrorMessage />;
-        setMessage(errorMessage);
+        setMessage(errorMessage[0].message);
```

Comment:
What happens if the previous line returns `<GenericUnknownErrorMessage />` ? This would error out, right?

Reply:
yes, you are right. This change was because of moving from the message component to the cds alert componet. I should have been using [0] on the getErrorMessageForRelayModernProblem.
addressed